extensions/wikia/ArticleComments/classes/ArticleComment.class.php

```diff
@@ -1538,7 +1536,7 @@ static public function isLoadingOnDemand() {
 	 */
 	static public function isMiniEditorEnabled() {
 		$app = F::app();
-		return $app->wg->EnableMiniEditorExtForArticleComments && $app->checkSkin( [ 'oasis', 'venus' ] );
+		return $app->wg->EnableMiniEditorExtForArticleComments && $app->checkSkin( [ 'oasis' ] );
```

Comment:
It's better to use array - https://github.com/Wikia/app/blob/dev/includes/wikia/nirvana/WikiaApp.class.php#L331